README.md
[![Build Status](https://travis-ci.org/akveo/ng2-admin.svg?branch=master)](https://travis-ci.org/akveo/ng2-admin)
[![Join the chat at https://gitter.im/ng2-admin/Lobby](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ng2-admin/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
[![Dependency Status](https://david-dm.org/akveo/ng2-admin/status.svg)](https://david-dm.org/akveo/ng2-admin)

# Admin panel framework based on Angular, Bootstrap 4 and Angular CLI

Admin template made with :heart:  by [Akveo team](http://akveo.com/). Follow us on [Twitter](https://twitter.com/akveo_inc) to get the latest news about this template first!

### Demo

<a target="_blank" href="http://akveo.com/ng2-admin/"><img src="http://i.imgur.com/QK9AzHj.jpg" width="600" alt="Sky Blue"/></a>

<a target="_blank" href="http://akveo.com/ng2-admin/">Live Demo</a>

## Angular 1.x version
Here you can find Angular 1.x based version: [Blur Admin](http://akveo.github.io/blur-admin/)
 
## Documentation
Installation, customization and other useful articles: https://akveo.github.io/ng2-admin/

## Based on
Angular 2+, Angular CLI, Bootstrap 4, and lots of awesome modules and plugins

## BrowserStack
This project runs its tests on multiple desktop and mobile browsers using [BrowserStack](http://www.browserstack.com).

<img src="https://cloud.githubusercontent.com/assets/131406/22254249/534d889e-e254-11e6-8427-a759fb23b7bd.png" height="40" />

## How can I support developers?
- Star our GitHub repo :star:
- Create pull requests, submit bugs, suggest new features or documentation updates :wrench:
- Follow us on [Twitter](https://twitter.com/akveo_inc) :feet:
- Like our page on [Facebook](https://www.facebook.com/akveo/) :thumbsup:

## Can I hire you guys?
Yes!  Visit [our homepage](http://akveo.com/) or simply leave us a message to [contact@akveo.com](mailto:contact@akveo.com). We will be happy to work with you!

## Features
* TypeScript
* Webpack
* Responsive layout
* High resolution
* Bootstrap 4 CSS Framework
* Sass
* Angular
* jQuery
* Charts (Chartist, Chart.js)
* Maps (Google, Leaflet, amMap)
* and many more!

## License
[MIT](LICENSE.txt) license.

## Special thanks to our awesome contributors!

[<img alt="nnixaa" src="https://avatars0.githubusercontent.com/u/230527?v=3&s=60" width="60">](https://github.com/nnixaa)[<img alt="lexzhukov" src="https://avatars0.githubusercontent.com/u/12192373?v=3&s=60" width="60">](https://github.com/lexzhukov)[<img alt="smartapant" src="https://avatars3.githubusercontent.com/u/3674348?v=3&s=60" width="60">](https://github.com/smartapant)[<img alt="Tibing" src="https://avatars2.githubusercontent.com/u/17410089?v=3&s=60" width="60">](https://github.com/Tibing)[<img alt="ESadouski" src="https://avatars3.githubusercontent.com/u/9255839?v=3&s=60" width="60">](https://github.com/ESadouski)[<img alt="hryb" src="https://avatars3.githubusercontent.com/u/24896429?v=3&s=60" width="60">](https://github.com/hryb)[<img alt="AlbertXingZhang" src="https://avatars3.githubusercontent.com/u/12808025?v=3&s=60" width="60">](https://github.com/AlbertXingZhang)[<img alt="iamdavidfrancis" src="https://avatars2.githubusercontent.com/u/539365?v=3&s=60" width="60">](https://github.com/iamdavidfrancis)[<img alt="Gelio" src="https://avatars3.githubusercontent.com/u/889383?v=3&s=60" width="60">](https://github.com/Gelio)[<img alt="ramsunvtech" src="https://avatars1.githubusercontent.com/u/1652629?v=3&s=60" width="60">](https://github.com/ramsunvtech)[<img alt="GRoguelon" src="https://avatars2.githubusercontent.com/u/309354?v=3&s=60" width="60">](https://github.com/GRoguelon)[<img alt="lugovsky" src="https://avatars3.githubusercontent.com/u/2788046?v=3&s=60" width="60">](https://github.com/lugovsky)[<img alt="kaleguy" src="https://avatars3.githubusercontent.com/u/1402176?v=3&s=60" width="60">](https://github.com/kaleguy)[<img alt="KostyaDanovsky" src="https://avatars3.githubusercontent.com/u/773632?v=3&s=60" width="60">](https://github.com/KostyaDanovsky)

[<img alt="RonnyRoos" src="https://avatars2.githubusercontent.com/u/6583791?v=3&s=60" width="60">](https://github.com/RonnyRoos)[<img alt="gavinzhou" src="https://avatars2.githubusercontent.com/u/240780?v=3&s=60" width="60">](https://github.com/gavinzhou)[<img alt="kbrandwijk" src="https://avatars2.githubusercontent.com/u/852069?v=3&s=60" width="60">](https://github.com/kbrandwijk)[<img alt="stacyakveo" src="https://avatars2.githubusercontent.com/u/27723447?v=3&s=60" width="60">](https://github.com/stacyakveo)[<img alt="Awk34" src="https://avatars1.githubusercontent.com/u/5545474?v=3&s=60" width="60">](https://github.com/Awk34)[<img alt="pritambaral" src="https://avatars1.githubusercontent.com/u/1378718?v=3&s=60" width="60">](https://github.com/pritambaral)[<img alt="shootermv" src="https://avatars3.githubusercontent.com/u/1336862?v=3&s=60" width="60">](https://github.com/shootermv)[<img alt="tgabi333" src="https://avatars0.githubusercontent.com/u/187022?v=3&s=60" width="60">](https://github.com/tgabi333)[<img alt="Kaizeras" src="https://avatars0.githubusercontent.com/u/6340189?v=3&s=60" width="60">](https://github.com/Kaizeras)[<img alt="mterwoord" src="https://avatars1.githubusercontent.com/u/114143?v=3&s=60" width="60">](https://github.com/mterwoord)[<img alt="michaeljota" src="https://avatars3.githubusercontent.com/u/10507776?v=3&s=60" width="60">](https://github.com/michaeljota)[<img alt="nstanard" src="https://avatars2.githubusercontent.com/u/4204806?v=3&s=60" width="60">](https://github.com/nstanard)[<img alt="myadhdoutlet" src="https://avatars0.githubusercontent.com/u/18659465?v=3&s=60" width="60">](https://github.com/myadhdoutlet)[<img alt="smith64fx" src="https://avatars0.githubusercontent.com/u/3246782?v=3&s=60" width="60">](https://github.com/smith64fx)

[<img alt="swaheed2" src="https://avatars1.githubusercontent.com/u/8592912?v=3&s=60" width="60">](https://github.com/swaheed2)[<img alt="susovan87" src="https://avatars0.githubusercontent.com/u/1897814?v=3&s=60" width="60">](https://github.com/susovan87)[<img alt="ThomasDupuis" src="https://avatars2.githubusercontent.com/u/1278762?v=3&s=60" width="60">](https://github.com/ThomasDupuis)[<img alt="turnerguo" src="https://avatars3.githubusercontent.com/u/633880?v=3&s=60" width="60">](https://github.com/turnerguo)[<img alt="anhuisunfei" src="https://avatars0.githubusercontent.com/u/5771096?v=3&s=60" width="60">](https://github.com/anhuisunfei)[<img alt="blackholegalaxy" src="https://avatars3.githubusercontent.com/u/11064123?v=3&s=60" width="60">](https://github.com/blackholegalaxy)[<img alt="bnayalivne" src="https://avatars0.githubusercontent.com/u/20381085?v=3&s=60" width="60">](https://github.com/bnayalivne)[<img alt="dimkk" src="https://avatars0.githubusercontent.com/u/4674044?v=3&s=60" width="60">](https://github.com/dimkk)[<img alt="geekow" src="https://avatars2.githubusercontent.com/u/12011794?v=3&s=60" width="60">](https://github.com/geekow)[<img alt="greenkeeperio-bot" src="https://avatars2.githubusercontent.com/u/14790466?v=3&s=60" width="60">](https://github.com/greenkeeperio-bot)[<img alt="pluswave" src="https://avatars0.githubusercontent.com/u/5653443?v=3&s=60" width="60">](https://github.com/pluswave)[<img alt="Guutong" src="https://avatars3.githubusercontent.com/u/10181975?v=3&s=60" width="60">](https://github.com/Guutong)[<img alt="Deilan" src="https://avatars1.githubusercontent.com/u/4777512?v=3&s=60" width="60">](https://github.com/Deilan)

### From akveo

Enjoy :metal:
We're always happy to receive your feedback!
"# ingresso-top-portal" 


CHANGELOG.md
<a name="1.0.0"></a>
# [1.0.0](https://github.com/akveo/ng2-admin/compare/v0.9.0...v1.0.0) (2017-04-27)


### Bug Fixes

* **build:** fix [#777](https://github.com/akveo/ng2-admin/issues/777) ([9bd7dbd](https://github.com/akveo/ng2-admin/commit/9bd7dbd))
* **build:** fix [#955](https://github.com/akveo/ng2-admin/issues/955) ([#961](https://github.com/akveo/ng2-admin/issues/961)) ([dd63535](https://github.com/akveo/ng2-admin/commit/dd63535))
* **build:** hopefully fix the prod & aot build errors ([#753](https://github.com/akveo/ng2-admin/issues/753)) ([7215b0b](https://github.com/akveo/ng2-admin/commit/7215b0b))
* **card:** fix [#962](https://github.com/akveo/ng2-admin/issues/962) ([#966](https://github.com/akveo/ng2-admin/issues/966)) ([72cb319](https://github.com/akveo/ng2-admin/commit/72cb319))
* **components:** add css pseudo-selector for dropdown arrow  ([#848](https://github.com/akveo/ng2-admin/issues/848)) ([1be434b](https://github.com/akveo/ng2-admin/commit/1be434b))
* **components:** add notification and message count from array length ([#893](https://github.com/akveo/ng2-admin/issues/893)) ([a8fe3f1](https://github.com/akveo/ng2-admin/commit/a8fe3f1))
* **dependencies:** fix [#689](https://github.com/akveo/ng2-admin/issues/689), [#688](https://github.com/akveo/ng2-admin/issues/688) ([d34e6e9](https://github.com/akveo/ng2-admin/commit/d34e6e9))
* **dependencies:** fix angular2-template-loader version ([2b25258](https://github.com/akveo/ng2-admin/commit/2b25258))
* **dependencies:** fix versions ([62810ae](https://github.com/akveo/ng2-admin/commit/62810ae))
* **dependencies:** fix versions ([5523029](https://github.com/akveo/ng2-admin/commit/5523029))
* **dependencies:** update dependencies, fix [#694](https://github.com/akveo/ng2-admin/issues/694), fix [#695](https://github.com/akveo/ng2-admin/issues/695) ([08de1f6](https://github.com/akveo/ng2-admin/commit/08de1f6))
* **docker:** fix [#712](https://github.com/akveo/ng2-admin/issues/712) ([cd7520e](https://github.com/akveo/ng2-admin/commit/cd7520e))
* **documents:** fix [#771](https://github.com/akveo/ng2-admin/issues/771) ([78c70ae](https://github.com/akveo/ng2-admin/commit/78c70ae))
* **editors:** fix an issue where ckeditor doesn't load on the demo ([0a4f40c](https://github.com/akveo/ng2-admin/commit/0a4f40c))
* **forms:** bad linking correction between label and input ([#886](https://github.com/akveo/ng2-admin/issues/886)) ([bb6ecd3](https://github.com/akveo/ng2-admin/commit/bb6ecd3))
* **maps:** fix [#813](https://github.com/akveo/ng2-admin/issues/813) ([b414eaf](https://github.com/akveo/ng2-admin/commit/b414eaf))
* **menu:** fix inactive menu item ([2468710](https://github.com/akveo/ng2-admin/commit/2468710))
* **package:** add the description fields ([0d77147](https://github.com/akveo/ng2-admin/commit/0d77147))
* **sidebar:** fix [#997](https://github.com/akveo/ng2-admin/issues/997) ([0a3c98e](https://github.com/akveo/ng2-admin/commit/0a3c98e))
* **style:** fix [#958](https://github.com/akveo/ng2-admin/issues/958) ([#963](https://github.com/akveo/ng2-admin/issues/963)) ([06d1b98](https://github.com/akveo/ng2-admin/commit/06d1b98))
* **tables:** fix [#1000](https://github.com/akveo/ng2-admin/issues/1000) ([#1001](https://github.com/akveo/ng2-admin/issues/1001)) ([4b98e0e](https://github.com/akveo/ng2-admin/commit/4b98e0e))
* **tables:** fix [#700](https://github.com/akveo/ng2-admin/issues/700) ([c85eae7](https://github.com/akveo/ng2-admin/commit/c85eae7))
* **tables:** fix [#704](https://github.com/akveo/ng2-admin/issues/704) ([0fbad7e](https://github.com/akveo/ng2-admin/commit/0fbad7e))
* **theme:** fix cards background of blur theme ([0528e2e](https://github.com/akveo/ng2-admin/commit/0528e2e))
* **ui:** fix NgbModal declaration ([49ff10e](https://github.com/akveo/ng2-admin/commit/49ff10e))
* **validator:** change email regex to: http://emailregex.com/ ([4470607](https://github.com/akveo/ng2-admin/commit/4470607))


### Features

* **angular-cli:** integrate angular-cli :tada: ([#989](https://github.com/akveo/ng2-admin/issues/989)) ([7932421](https://github.com/akveo/ng2-admin/commit/7932421))
* **dependencies:** update yarn.lock file, fix debugging ([cc97bf1](https://github.com/akveo/ng2-admin/commit/cc97bf1))
* **layouts:** add file uploader ([#842](https://github.com/akveo/ng2-admin/issues/842)) ([7d03461](https://github.com/akveo/ng2-admin/commit/7d03461))
* **localization:** implement the localization using ngx-translate ([#830](https://github.com/akveo/ng2-admin/issues/830)) ([627d62d](https://github.com/akveo/ng2-admin/commit/627d62d))
* **login:** add the locale changes ([#993](https://github.com/akveo/ng2-admin/issues/993)) ([e033edf](https://github.com/akveo/ng2-admin/commit/e033edf))
* **test:** add testing infrastructure: karma, jasmine and chrome ([#982](https://github.com/akveo/ng2-admin/issues/982)) ([4420048](https://github.com/akveo/ng2-admin/commit/4420048))


### BREAKING CHANGES

* The webpack version of ng2-admin was moved to webpack [branch](https://github.com/akveo/ng2-admin/tree/webpack).
* We dropped the electron support as, unfortunately, angular-cli doesn't have the official support of electron. Please consider using preview 'webpack' (link to webpack branch) version manually.


### How to migrate

1. Pull the latest ng2-admin version from **https://github.com/akveo/ng2-admin.git**;
2. Copy your **src/app, src/assets, src/custom-typings.d.ts** from webpack version into **src** folder in cli version;
3. Add each of your required modules and styles into **.angular-cli.json**;
5. Change paths to file(images, fonts, ets);
6. Add **import * as $ from 'jquery';** into **app.component.ts** in cli version;

src/app/pages/pages.menu.ts
export const PAGES_MENU = [
  {
    path: 'pages',
    children: [
      {
        path: 'dashboard',
        data: {
          menu: {
            title: 'general.menu.dashboard',
            icon: 'ion-android-home',
            selected: false,
            expanded: false,
            order: 0
          }
        }
      },
      {
        path: 'editors',
        data: {
          menu: {
            title: 'general.menu.editors',
            icon: 'ion-edit',
            selected: false,
            expanded: false,
            order: 100,
          }
        },
        children: [
          {
            path: 'ckeditor',
            data: {
              menu: {
                title: 'general.menu.ck_editor',
              }
            }
          }
        ]
      },
      {
        path: 'components',
        data: {
          menu: {
            title: 'general.menu.components',
            icon: 'ion-gear-a',
            selected: false,
            expanded: false,
            order: 250,
          }
        },
        children: [
          {
            path: 'treeview',
            data: {
              menu: {
                title: 'general.menu.tree_view',
              }
            }
          }
        ]
      },
      {
        path: 'charts',
        data: {
          menu: {
            title: 'general.menu.charts',
            icon: 'ion-stats-bars',
            selected: false,
            expanded: false,
            order: 200,
          }
        },
        children: [
          {
            path: 'chartist-js',
            data: {
              menu: {
                title: 'general.menu.chartist_js',
              }
            }
          }
        ]
      },
      {
        path: 'ui',
        data: {
          menu: {
            title: 'general.menu.ui_features',
            icon: 'ion-android-laptop',
            selected: false,
            expanded: false,
            order: 300,
          }
        },
        children: [
          {
            path: 'typography',
            data: {
              menu: {
                title: 'general.menu.typography',
              }
            }
          },
          {
            path: 'buttons',
            data: {
              menu: {
                title: 'general.menu.buttons',
              }
            }
          },
          {
            path: 'icons',
            data: {
              menu: {
                title: 'general.menu.icons',
              }
            }
          },
          {
            path: 'modals',
            data: {
              menu: {
                title: 'general.menu.modals',
              }
            }
          },
          {
            path: 'slim',
            data: {
              menu: {
                title: 'Slim loading bar',
              }
            }
          },
          {
            path: 'grid',
            data: {
              menu: {
                title: 'general.menu.grid',
              }
            }
          },
        ]
      },
      {
        path: 'forms',
        data: {
          menu: {
            title: 'general.menu.form_elements',
            icon: 'ion-compose',
            selected: false,
            expanded: false,
            order: 400,
          }
        },
        children: [
          {
            path: 'inputs',
            data: {
              menu: {
                title: 'general.menu.form_inputs',
              }
            }
          },
          {
            path: 'layouts',
            data: {
              menu: {
                title: 'general.menu.form_layouts',
              }
            }
          }
        ]
      },
      {
        path: 'tables',
        data: {
          menu: {
            title: 'general.menu.tables',
            icon: 'ion-grid',
            selected: false,
            expanded: false,
            order: 500,
          }
        },
        children: [
          {
            path: 'basictables',
            data: {
              menu: {
                title: 'general.menu.basic_tables',
              }
            }
          },
          {
            path: 'smarttables',
            data: {
              menu: {
                title: 'general.menu.smart_tables',
              }
            }
          },
          {
            path: 'datatables',
            data: {
              menu: {
                title: 'Data Tables',
              }
            }
          },
           {
             path: 'hottables',
             data: {
               menu: {
                 title: 'Hot Tables',
               }
             }
           }
        ]
      },
      {
        path: 'maps',
        data: {
          menu: {
            title: 'general.menu.maps',
            icon: 'ion-ios-location-outline',
            selected: false,
            expanded: false,
            order: 600,
          }
        },
        children: [
          {
            path: 'googlemaps',
            data: {
              menu: {
                title: 'general.menu.google_maps',
              }
            }
          },
          {
            path: 'leafletmaps',
            data: {
              menu: {
                title: 'general.menu.leaflet_maps',
              }
            }
          },
          {
            path: 'bubblemaps',
            data: {
              menu: {
                title: 'general.menu.bubble_maps',
              }
            }
          },
          {
            path: 'linemaps',
            data: {
              menu: {
                title: 'general.menu.line_maps',
              }
            }
          }
        ]
      },
      {
        path: '',
        data: {
          menu: {
            title: 'general.menu.pages',
            icon: 'ion-document',
            selected: false,
            expanded: false,
            order: 650,
          }
        },
        children: [
          {
            path: ['Cliente/login'],
            data: {
              menu: {
                title: 'general.menu.login'
              }
            }
          },
          {
            path: ['Cliente/register'],
            data: {
              menu: {
                title: 'general.menu.register'
              }
            }
          }
        ]
      },
      {
        path: '',
        data: {
          menu: {
            title: 'general.menu.menu_level_1',
            icon: 'ion-ios-more',
            selected: false,
            expanded: false,
            order: 700,
          }
        },
        children: [
          {
            path: '',
            data: {
              menu: {
                title: 'general.menu.menu_level_1_1',
                url: '#'
              }
            }
          },
          {
            path: '',
            data: {
              menu: {
                title: 'general.menu.menu_level_1_2',
                url: '#'
              }
            },
            children: [
              {
                path: '',
                data: {
                  menu: {
                    title: 'general.menu.menu_level_1_2_1',
                    url: '#'
                  }
                }
              }
            ]
          }
        ]
      },
      {
        path: '',
        data: {
          menu: {
            title: 'general.menu.external_link',
            url: 'http://akveo.com',
            icon: 'ion-android-exit',
            order: 800,
            target: '_blank'
          }
        }
      }
    ]
  }
];
